README.md

```diff
@@ -1358,7 +1369,8 @@ register as a public company earlier.
 
 🚧 Why? More links on history of RSUs and Facebook story?
 
-🄳 [**Restricted stock units**](http://www.investopedia.com/terms/r/restricted-stock-unit.asp) (**RSUs**)
+🄳
+[**Restricted stock units (RSUs)**](http://www.investopedia.com/terms/r/restricted-stock-unit.asp)
```

Comment:
FYI this was an incorrectly formatted definition as the abbrev should be inside the link.